spec/views/home/dashboard.html.erb_spec.rb

```diff
@@ -2,8 +2,12 @@
 
 RSpec.describe 'home/dashboard', type: :view do
   let(:users) { FactoryBot.create_list :user, 3, current_status: User.filter_status }
+  let(:room_messages) { FactoryBot.create_list :room_message, 3, room: Room.find(Room::GLOBAL_CHAT_ID) }
```

Comment:
Why did you define a scope if you never use it? ^^

Reply:
Because we only defined it after everything else